packages/block-library/src/cover/editor.scss

```diff
@@ -1,9 +1,4 @@
 .wp-block-cover {
-	/* Extra specificity needed because the reset.css applied in the editor context is overriding this rule. */
-	.editor-styles-wrapper & {
-		box-sizing: border-box;
-	}
```

Comment:
Removed because there's a rule in the style.scss that does the same thing.

It also doesn't seem to override anything in the reset.css, maybe that was refactored after this was introduced.